firebase-vertexai/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 * [fixed] Added support for new values sent by the server for `FinishReason` and `BlockReason`.
 * [changed] Added support for modality-based token count. (#6658)
+* [feature] Added support for generating images with Imagen models.
```

Comment:
As a heads up, this was the CHANGELOG entry I put in the Swift SDK. Consider adding the preview notes if it makes sense in the context of the Android release notes generation process: https://github.com/firebase/firebase-ios-sdk/blob/37b6456bdd9dc3e170d3eb0b9cf0cd9408f1a747/FirebaseVertexAI/CHANGELOG.md?plain=1#L2C1-L7C31

Reply:
Ack